File: Practice/codeptit/basic/t1289/t1289.cpp
#include <bits/stdc++.h>
using namespace std;
int gcd(int a, int b) {
    while (b != 0) {
        int r = a % b;
        a = b;
        b = r;

    }
    return a;
}
long long lcm(int a, int b) {
    long long res = a * b / gcd(a, b);
    return res;
}
int main () {
    int t;
    cin >> t;
    while (t--) {
        int a, b;
        cin >> a >> b;
        cout << gcd(a, b) << " " << lcm(a, b) << endl;
    }
}

File: Practice/codeptit/basic/t1289/t1289.py
def gcd(a, b):
    while b != 0:
        r = a % b
        a = b
        b = r
    return a
def lcm(a, b):
    return a * b / gcd(a, b)
test = int(input())
while test > 0:
    a, b = map(int, input().split())
    print(int(gcd(a, b)), int(lcm(a, b)))
    test -= 1